lib/logmsg/tests/test_tags.c

```diff
@@ -159,55 +141,66 @@ test_msg_tags(void)
 
 }
 
-void
-test_filters(gboolean not)
+ParameterizedTestParameters(tags, test_filters)
+{
+  const gboolean not[] =
+  {
+    TRUE,
+    FALSE
+  };
+  return cr_make_param_array(const gboolean, not, sizeof(not) / sizeof(const gboolean));
+}
+
+ParameterizedTest(const gboolean *not, tags, test_filters)
```

Comment:
Optional: I am not really sure that parameterizing true/false would it make any better (not really). Probably this should be a different testcases, as understanding such assert `(i % 3 == 1) ^ filter_expr_eval(f, msg)) ^ not[0]` should not be anybodies task.
This is optional as it was not okay before your PR, and this PR did not make it worse.

Reply:
Should I then change them to two different cases?

Reply:
I would prefer you do, but if you won't that is not going to block the PR.

Reply:
@parthw1 Would you like to refactor the test case as @Kokan suggested, or would you just stop here with this PR?

Reply:
@MrAnno I will refactor it into two different cases and push changes again.

Reply:
Thank you! :+1:

Reply:
@MrAnno I have pushed changes. Please check.